core/auth/src/main/java/software/amazon/awssdk/auth/credentials/AwsBasicCredentials.java

```diff
@@ -36,17 +40,31 @@
  */
 @Immutable
 @SdkPublicApi
-public final class AwsBasicCredentials implements AwsCredentials {
+public final class AwsBasicCredentials implements AwsCredentials,
+                                                  ToCopyableBuilder<AwsBasicCredentials.Builder, AwsBasicCredentials> {
     /**
      * A set of AWS credentials without an access key or secret access key, indicating that anonymous access should be used.
-     *
-     * This should be accessed via {@link AnonymousCredentialsProvider#resolveCredentials()}.
      */
+    // TODO(sra-identity-and-auth): Check if this static member can be removed after cleanup
```

Comment:
Because it's not used anymore for AnonymousCredentialsProvider through this change, but it's used on some logic paths for credential resolution for pre-SRA request pipelines. The SRA cleanup task should also address this part of the code.

Reply:
okay , Thanks